src/machines/systemIO/hooks.ts

```diff
@@ -0,0 +1,21 @@
+import { systemIOActor } from '@src/lib/singletons'
+import { useSelector } from '@xstate/react'
+export const useRequestedProjectName = () =>
```

Comment:
wondering how we police this? (and other conventions you've got here.)

Vigilence of reviewers, or is there something more automated we can do that's not too much dev time sunk into it?

Reply:
Yeah I mean the convention of putting hooks into this file (and others), probably can be just played by ear.

I do like the idea of lint rule, so that robots are doing the policing, but don't like the time spend setting it up and maintaining it, for our team size at least.